tools/isobuild/bundler.js

```diff
@@ -1522,11 +1522,17 @@ class ClientTarget extends Target {
     if (this.arch === 'web.cordova') {
       const { WebAppHashing } =
         loadIsopacket('cordova-support')['webapp-hashing'];
+
       const cordovaCompatibilityVersions =
         _.object(_.map(CORDOVA_PLATFORM_VERSIONS, (version, platform) => {
-          const hash = WebAppHashing.calculateCordovaCompatibilityHash(
-            version,
-            this.cordovaDependencies);
+          let cordovaDependencies = Object.assign({}, this.cordovaDependencies);
+          let pluginsExcludedFromCompatibilityHash = process.env.CORDOVA_COMPATIBILITY_VERSION_EXCLUDE || '';
```

Comment:
I realize this variable is already long, but we should prefix (new) environment variables with `METEOR_`.

Also, I think "version" makes sense for the other environment variable, but maybe this should be "hash" (since it's being excluded from the `calculateCordovaCompatibilityHash` function)? (Also, it's shorter!)

Maybe `METEOR_CORDOVA_COMPAT_HASH_EXCLUDE`?